internal/postgresql/postgresql.go

```diff
@@ -61,6 +61,7 @@ type PGManager interface {
 
 type Manager struct {
 	pgBinPath             string
+	disableAlterSystem    bool
```

Comment:
Can we pull this line to the bottom of the struct fields please, under requestTimeout? The fields are vaguely ordered in importance, which is why dataDir and parameters etc are up top. This type of flag is really optional, and is best placed as something developers see last.